index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
I see.
And why not allowing users to set a key named `onSchema` from `config(app, Store)` with a function as a value? We can call it after Model creation.
Like `association(model)` in trailpack-sequelize that it's called for handling associations after Model creation but returned by `config(app, Store)`